command.go

```diff
@@ -21,13 +21,23 @@ var (
 	ErrUnknownShell = errors.New("Unknown shell")
 )
 
+var (
+	HelpRequested bool
```

Comment:
I like making this a package variable over how I have been doing it as a member on the struct I'm populating.